security-response.md

```diff
@@ -0,0 +1,107 @@
+encrypted message.
+
+[gh-organization]: https://github.com/open-telemetry
+
+### Public Disclosure Processes
+
+If you know of a publicly disclosed security vulnerability please IMMEDIATELY
+email
+[cncf-opentelemetry-tc@lists.cncf.io](mailto:cncf-opentelemetry-tc@lists.cncf.io)
+to inform the Security Response Committee (SRC) about the vulnerability so they
+may start the patch, release, and communication process.
+
+If possible the TC will ask the person making the public report if the issue can
+be handled via a private disclosure process. If the reporter denies the request,
+the TC will move swiftly with the fix and release process. In extreme cases you
+can ask GitHub to delete the issue but this generally isn't necessary and is
+unlikely to make a public disclosure less damaging.
+
+## Patch, Release, and Public Communication
+
+For each vulnerability a member of the TC will volunteer to lead coordination
+with the relevant repo owners, and is responsible for sending disclosure
+emails to the rest of the community. This TC member will be referred to as the Fix Lead.
+
+The role of Fix Lead should rotate round-robin across the TC.
+
+All of the timelines below are suggestions and assume a Private Disclosure.
+The Fix Lead drives the schedule using their best judgment based on severity,
+development time, and release manager feedback. If the Fix Lead is dealing with
+a Public Disclosure all timelines become ASAP. If the fix relies on another
+upstream project's disclosure timeline, that will adjust the process as well.
+We will work with the upstream project to fit their timeline and best protect
+our users.
+
+### Fix Team Organization
+
+The Fix Team is made up of the Fix Team Lead and the relevant repo maintainers / contributors.
+
+### Fix Development Process
+
+These steps should be completed within the 1-7 days of Disclosure.
+
+- The Fix Lead and the Fix Team will create a [CVSS](https://www.first.org/cvss/specification-document) score using the [CVSS Calculator](https://www.first.org/cvss/calculator/3.0). The Fix Lead makes the final call on the calculated risk; it is better to move quickly than make the perfect assessment.
+- The Fix Lead will request a CVE from GitHub.
+- The Fix Team will notify the Fix Lead that work on the fix branch is complete once there are LGTMs on all commits in the private repo.
+
+### Fix Disclosure Process
+
+OTel relies on GitHub tooling to notify the affected repositories and publish a security advisory. GitHub will publish the CVE to the CVE List, broadcast the Security Advisory via the GitHub Advisory Database, and send security alerts to all repositores that use the package and have alerts on.
+
+#### Fix Release Day
+
+The Fix Team as repo owners will release an updated version after confirming approval with the Fix Lead.
+
+## Retrospective
+
+### OTel Retrospective Policy
+
+?
+The assigned TC member coordinates the retrospective?
+
+### Kubernetes Policy
```

Comment:
Is this here only for reference?

Reply:
More of do we want something similar but will remove for now.